cluster/libvirt-coreos/openssl.cnf

```diff
@@ -0,0 +1,15 @@
+[req]
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+[req_distinguished_name]
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = kubernetes
+DNS.2 = kubernetes.default
+DNS.3 = kubernetes.default.svc
+DNS.4 = kubernetes.default.svc.cluster.local
+IP.1 = 10.11.0.1
```

Comment:
Since the service range can be configured (https://github.com/kubernetes/kubernetes/blob/master/cluster/libvirt-coreos/config-default.sh#L49) it would be best to just take the first IP out of the range from the environment variable (which is a PITA, I know, but I think we do it elsewhere in bash that you could copy/paste).